app/Http/Controllers/Entity/InventoryController.php

```diff
@@ -156,6 +155,28 @@ public function update(UpdateInventory $request, Campaign $campaign, Entity $ent
             ]));
     }
 
+    /**
```

Comment:
Would be cleaner to have this be it's own mini controller rather than added on top of the existing one :)